### ISSUE TITLE
Build a macOS PyInstaller binary of Datasette plus the app-support plugin

### ISSUE THREAD

**Issue by simonw** [simonw/datasette-app#17]
Split from #7.

**Comment by simonw** [comment]
I'm going to try my recipe from https://github.com/simonw/datasette/issues/93#issuecomment-754219002

**Comment by simonw** [comment]
I used the pre-existing recipe:
```
pip install wheel
pip install datasette pyinstaller

BASE=$(python -c 'import os; print(os.path.dirname(__import__("datasette").__file__))') \
    pyinstaller -F \
        --add-data "$BASE/templates:datasette/templates" \
        --add-data "$BASE/static:datasette/static" \
        --hidden-import datasette.publish \
        --hidden-import datasette.publish.heroku \
        --hidden-import datasette.publish.cloudrun \
        --hidden-import datasette.facets \
        --hidden-import datasette.sql_functions \
        --hidden-import datasette.actor_auth_cookie \
        --hidden-import datasette.default_permissions \
        --hidden-import datasette.default_magic_parameters \
        --hidden-import datasette.blob_renderer \
        --hidden-import datasette.default_menu_links \
        --hidden-import uvicorn \
        --hidden-import uvicorn.logging \
        --hidden-import uvicorn.loops \
        --hidden-import uvicorn.loops.auto \
        --hidden-import uvicorn.protocols \
        --hidden-import uvicorn.protocols.http \
        --hidden-import uvicorn.protocols.http.auto \
        --hidden-import uvicorn.protocols.websockets \
        --hidden-import uvicorn.protocols.websockets.auto \
        --hidden-import uvicorn.lifespan \
        --hidden-import uvicorn.lifespan.on \
        $(which datasette)
```
It sort-of worked: `dist/datasette` is a binary that starts. But... it takes about 9 seconds to start the server!

https://stackoverflow.com/questions/9469932/app-created-with-pyinstaller-has-a-slow-startup suggests that this might be because I am using "one file" mode, with the `-F` flag (a shortcut for `--onefile`).

So I swapped that out for `--onedir` - but that's not finding and copying in the templates. Need to investigate more.

**Comment by simonw** [comment]
Closed in favour of #25.
